package.json

```diff
@@ -41,6 +41,7 @@
     "passport": "^0.3.2",
     "passport-local": "^1.0.0",
     "pg": "^4.5.1",
+    "pg-pool": "^1.4.3",
```

Comment:
Create client is used for some one off db connections. Not for anything user facing